x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/helpers.ts

```diff
@@ -163,6 +164,9 @@ export const getExampleText = (example: string | number | null | undefined): str
 /** Returns `true` if the escape key was pressed */
 export const isEscape = (event: React.KeyboardEvent): boolean => event.key === 'Escape';
 
+export const getShowDescriptionColumn = (fieldItems: BrowserFieldItem[]) =>
+  fieldItems.some((fieldItem) => !!fieldItem.description?.trim());
```

Comment:
as it might be a very large number of fields and in order to avoid running multiple times through this array, what about refactoring the `getFieldItems` fn in `x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx` so it also let you know somehow if there are descriptions or not?